### DISCUSSION TITLE
Virtual VulkanSceneGraph-1.0 Release Party

### DISCUSSION THREAD

**Discussion by robertosfield** [vsg-dev/VulkanSceneGraph#574] [General]
Hi All,

I have scheduled a 1hr Zoom call for 8pm GMT this evening, Sunday 13th Nov. 2022.

To join Zoom Meeting: [https://us02web.zoom.us/j/88463231951](https://us02web.zoom.us/j/88463231951)

As long as no show stoppers appear today  that's when I'll tag 1.0.0 as well.  Everyone is welcome to attend.  I plan to record the session and post it to my youtube account if it looks suitable for sharing!

I would really appreciate hearing from others in the community on how you are using the VulkanSceneGraph and what your plans are. During the call I can hand over the mike/screenshare to folks that want to present their projects/work.  If you'd like to present something let me know beforehand  so I can have a rough idea of how to schedule the different speakers.

I look forward to meeting you all later today :-)

Robert.

**Comment by robertosfield** [comment]
Hi All,

Just finished the Zoom call, and mid way through ran the release script to officially tag VulkanSceneGraph-1.0.0, vsgXchange-1.0.0 and vsgExamples-1.0.0:

    https://github.com/vsg-dev/VulkanSceneGraph/releases/tag/VulkanSceneGraph-1.0.0
    https://github.com/vsg-dev/vsgXchange/releases/tag/vsgXchange-1.0.0
    https://github.com/vsg-dev/vsgExamples/releases/tag/vsgExamples-1.0.0

Also announced during call is vsgPoints, and open source library that renders large numbers of points (tested up to 450 million so far), it's very early days for this project so don't expect it to be feature rich at this point.
    https://github.com/robertosfield/vsgPoints

Thanks to everyone who's companies help fund development, engineers that have contributed code, fixes and time testing, and testing, testing. All invaluable and much appreciated :-)

Cheers,
Robert Osfield.
VulkanSceneGraph Project Lead

**Comment by kannode** [comment]
Sorry I missed the party. Will look for zoom video on youtube.

**Comment by robertosfield** [comment]
On Thu, 17 Nov 2022 at 18:20, kannode ***@***.***> wrote:

> Sorry I missed the party. Will look for zoom video on youtube.
>
Please don't expect a slick presentation :-)

Tomorrow I'll have a bash at uploading the video to youtube.

**Reply by patstevenson8** [reply]
Just established my GitHub presence and am very interested in contributing to this effort.  I've been in graphics and visualization since my SGI IRIS GL days!  I will look for the launch party video.  Meanwhile, I would like to try to duplicate some things from OSG, especially osgEarth for vsg.  May be a bit much to bite off, so was also wondering about something smaller to get my feet wet?

**Reply by sbrkopac** [reply]
Welcome, Pat! I got my feet wet by taking some of the osgExamples and converting them to vsgExamples. I've also taken the examples from [Sascha Willems](https://www.saschawillems.de/creations/vulkan-examples/) and attempted to move those over. Just some thoughts to get started.

**Comment by robertosfield** [comment]
Hi Pat,

On Thu, 17 Nov 2022 at 22:27, Pat Stevenson ***@***.***>
wrote:

> Just established my GitHub presence and am very interested in contributing
> to this effort. I've been in graphics and visualization since my SGI IRIS
> GL days!
>
That's the era I began coding in, in 92 learnt IrisGL then OpenGL, fast
forward 30 years and I'm playing with Vulkan.


> I will look for the launch party video. Meanwhile, I would like to try to
> duplicate some things from OSG, especially osgEarth for vsg. May be a bit
> much to bite off, so was also wondering about something smaller to get my
> feet wet?
>

There is vsgEarth if you're curious about osgEarth->VSG.  Tim Moore who did
the work on vsgEarth talks about it on the Zoom call.

Cheers,
Robert.

**Comment by ghost** [comment]
I have uploaded video, not polished but hopefully helpful:

https://www.youtube.com/watch?v=mqiCrbQkXd8
